src/components/ebay-menu-button/examples/20-prefix-id/template.marko

```diff
@@ -0,0 +1,18 @@
+class {
+    onCreate() {
+        this.state = { buttonText: 'Select An Option' };
```

Comment:
Very last thing I promise, but can we have the first value (i.e. Name) instead of 'Select an Option' as the initial text.